src/NServiceBus.RabbitMQ/Configuration/ConnectionConfiguration.cs

```diff
@@ -62,14 +149,51 @@ void SetDefaultClientProperties(string endpointName)
 
             var hostname = RuntimeEnvironment.MachineName;
 
-            ClientProperties.Add("client_api", "NServiceBus");
-            ClientProperties.Add("nservicebus_version", nsbFileVersion);
-            ClientProperties.Add("nservicebus.rabbitmq_version", rabbitMQFileVersion);
-            ClientProperties.Add("application", applicationName);
-            ClientProperties.Add("application_location", applicationPath);
-            ClientProperties.Add("machine_name", hostname);
-            ClientProperties.Add("user", UserName);
-            ClientProperties.Add("endpoint_name", endpointName);
+            var clientProperties = new Dictionary<string, string>
+            {
+                { "client_api", "NServiceBus" },
+                { "nservicebus_version", nsbFileVersion },
+                { "nservicebus.rabbitmq_version", rabbitMQFileVersion },
+                { "application", applicationName },
+                { "application_location", applicationPath },
+                { "machine_name", hostname },
+                { "user", userName },
+                { "endpoint_name", endpointName },
+            };
+
+            if (invalidOptionsMessage.Length > 0)
+            {
+                var message = invalidOptionsMessage.ToString().TrimEnd('\r', '\n');
+
+                Logger.Error(message);
```

Comment:
Exactly, or if not, it will go all the way up the stack and kill the process as unhandled. Regardless, logging it is not concern of this class. I'll add another commit to remove this.